### ISSUE TITLE
Accessibility: paragraph spacing override

### ISSUE THREAD

**Issue by henrikaskj** [SemanticComputing/sampo-ui#47]
Success criterion 1.4.12 in WCAG 2.1 states that one should be able to override the bottom margins of paragraphs without loss of content, up to 2em.
In Sampo-UI this results in a clipped element in the facets:
![image](https://user-images.githubusercontent.com/22298175/214857322-1fdf815f-366c-44ce-b0a4-272521a247bb.png)
[](url)

# Suggested solution
Replace the \<p\> elements, perhaps by using a MUI typography variant other than body1
